includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp-cached-data.php

```diff
@@ -557,7 +557,7 @@ public static function fetch_lists( $limit = null ) {
 			__( 'Error retrieving Mailchimp lists.', 'newspack_newsletters' )
 		);
 		if ( is_wp_error( $lists_response ) || empty( $lists_response['lists'] ) ) {
-			Newspack_Newsletters_Logger::log( 'Mailchimp cache: Error refreshing cache: ' . ( $lists_response->getMessage() ?? __( 'Error retrieving Mailchimp lists.', 'newspack_newsletters' ) ) );
+			Newspack_Newsletters_Logger::log( 'Mailchimp cache: Error refreshing cache: ' . ( is_wp_error( $lists_response ) ? $lists_response->getMessage() : __( 'Error retrieving Mailchimp lists.', 'newspack_newsletters' ) ) );
```

Comment:
I don't think this is a 'WP_Error` object. WP_Error method is `get_message`, not `getMessage`. I think this is something from the Mailchimp lib

Reply:
No wait, this 63a9178

Reply:
Argh, cb62cfe